### PR TITLE
chore: bump gear-dlmalloc for free_range support

### PR DESCRIPTION
**Pull request by holykol**
Bump allocator to version that uses new `free_range` call.

@gear-tech/dev

### CODE REVIEW

**Review by shamilsan — APPROVED** [review]
Need to add a `0.1.5` tag there too: https://github.com/gear-tech/dlmalloc-rust/tags

**Review by clearloop — APPROVED** [review]

**Comment by breathx** [comment]
> Need to add a `0.1.5` tag there too: https://github.com/gear-tech/dlmalloc-rust/tags

We've never tagged it before. Should we?

**Review by breathx — APPROVED** [review]

**Comment by holykol** [comment]
@breathx
> We've never tagged it before. Should we?

I've already published `0.1.5` on crates.io, so tagging just makes it consistent. If anything, it should have been a major version change because it technically breaks backwards compatibility by relying on a new syscall

File: Cargo.toml [hunk 1]
```diff
@@ -166,8 +166,8 @@ tempfile = "3.8.1"
 
 # Published deps
 #
-# https://github.com/gear-tech/dlmalloc-rust/tree/v0.1.4
-dlmalloc = { package = "gear-dlmalloc", version = "0.1.4" }
+# https://github.com/gear-tech/dlmalloc-rust/tree/v0.1.5
+dlmalloc = { package = "gear-dlmalloc", version = "0.1.5" }
```
Comment (shamilsan) [inline Cargo.toml:170]: Bump version to 0.2.0 due to breaking changes (see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)

**Review by shamilsan — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (holykol) [reply Cargo.toml]: Done

File: Cargo.toml [hunk 2]
```diff
@@ -166,8 +166,8 @@ tempfile = "3.8.1"
 
 # Published deps
 #
-# https://github.com/gear-tech/dlmalloc-rust/tree/v0.1.4
-dlmalloc = { package = "gear-dlmalloc", version = "0.1.4" }
+# https://github.com/gear-tech/dlmalloc-rust/tree/v0.1.5
```
Comment (shamilsan) [inline Cargo.toml:169]: Please change this link too
```suggestion
# https://github.com/gear-tech/dlmalloc-rust/tree/0.2.0
```

**Review by shamilsan — APPROVED** [review]

**Review by breathx — APPROVED** [review]

File: Cargo.toml (hunk 2 above)
Comment (holykol) [reply Cargo.toml]: thanks

**Pull request merged**